Lib/fontbakery/codetesting.py

```diff
@@ -155,7 +149,34 @@ def get_args(self, condition_overrides=None):
                 args[k] = list(args[k])
         return args
 
-    def __call__(self,**condition_overrides):
+    def __call__(self, values, condition_overrides={}):
+        from fontTools.ttLib import TTFont
+        if isinstance(values, str):
+            values = {'font': values,
+                      'fonts': [values]}
+        elif isinstance(values, TTFont):
+            values = {'font': values.reader.file.name,
+                      'fonts': [values.reader.file.name],
+                      'ttFont': values,
+                      'ttFonts': [values]}
```

Comment:
`ttFonts` as `values` won't have any effect.it's defined as a `derrived_iterable` not as a `value`.

Reply:
Actually, `font` is neither a value, it's an item from the `iterable` of `fonts` and `ttfont` is not  a value as well, it's a `condition` of `font`